src/NUnitFramework/mock-assembly/mock-assembly-netstandard13.csproj

```diff
@@ -63,14 +64,17 @@
     <None Include="..\..\nunit.snk">
       <Link>nunit.snk</Link>
     </None>
-    <None Include="mock-assembly-portable.project.json" />
+    <None Include="mock-assembly-netstandard13.project.json" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
```

Comment:
No, we do want these. We are using the old `CSPROJ` portable format, not the new .NET Core one, so the `project.json` files are still needed. This allows us to build in VS2015. At some point I would like to upgrade all of our projects to 2017 and the new project so we can use multi-targeting, but that is a ways out.